src/wp-admin/js/inline-edit-post.js

```diff
@@ -240,7 +297,14 @@ inlineEditPost = {
 		return false;
 	},
 
-	// Ajax saving is only for Quick Edit.
+
+	/**
+	 * Saves the changes made in the quick edit window to the post.
+	 * Ajax saving is only for Quick Edit and not for bulk edit.
+	 *
+	 * @param   {int} id The id for the post that has been changed.
```

Comment:
This param also doesn't necessarily have to an int. It could also be an object (again) 😅  😢 .